packages/api/src/decorator/private/activityDecoratorRequest.ts

```diff
@@ -0,0 +1,6 @@
+type ActivityDecoratorRequestType = {
+  from: 'user' | 'bot' | 'channel';
```

Comment:
To handle cases that the `from` is not valid.

```suggestion
  from: 'bot' | 'channel' | `user` | undefined;
```